src/app/application/components/route-view/RouteView.vue

```diff
@@ -71,6 +86,31 @@ if (!hasParent) {
   provide('route-view-parent', routeView)
 }
 
+const iParent: ImmediateParent | undefined = inject('route-view-immediate-parent', undefined)
+
+const immediateParent: ImmediateParent = {
+  addChild: (module, sym) => {
+    children.value.push(module)
+    if (typeof iParent !== 'undefined') {
+      // @ts-ignore
+      iParent.addChild(module, sym)
```

Comment:
Typescript tells me this can `never` happen, javascript tells me otherwise.

Maybe I'm doing something wrong, but I didn't want to spend to much time looking into it right now, seeing as I'm coming back here after the PR is merged, but if its a simple "oh you just need to...." lmk